### PR TITLE
Fix bugs in woq algorithms and llama example

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

bug fix
API changed or not: no

## Description

Fix bugs in woq algorithms and llama example.
- Fix config bug in AWQ algorithm
- Fix parameter bug in GPTQ algorithm 
- Remove ITREX dependency in llama example

## How has this PR been tested?

local test

## Dependency Change?

no

### CODE REVIEW

File: examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
```
Comment (justinchuby) [inline examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py:3]:
```suggestion
```

nit: just flying by

File: examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/evaluator.py [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+    predict_only: bool = False,
+    random_seed: int = 0,
+    numpy_random_seed: int = 1234,
+    torch_random_seed: int = 1234,
+    user_model: Optional[object] = None,
+    tokenizer: Optional[object] = None,
+):
+    """Instantiate and evaluate a model on a list of tasks.
+
+    :param model: Union[str, LM]
```
Comment (justinchuby) [inline examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/evaluator.py:81]: nit: prefer google style docstrings

File: examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py (hunk 1 above)
Comment (yuwenzho) [reply examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py]: Fixed. Thanks for your review.

File: examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/evaluator.py (hunk 2 above)
Comment (yuwenzho) [reply examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/evaluator.py]: Fixed. Thanks for your review.

**Review by chensuyue — APPROVED** [review]

File: examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Refer from https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/transformers/llm/evaluation/lm_eval
+
+from .accuracy import cli_evaluate as evaluate
```
Comment (mengniwang95) [inline examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py:17]: https://gist.github.com/justinchuby/9085242a53158f2fd7ae7aa650e55ee3 prefer not using relative imports

Comment (yuwenzho) [reply examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py]: Fixed. Thanks for review.

**Review by hshen14 — APPROVED** [review]

**Pull request merged**